client/src/app/site/agenda/components/agenda-list/agenda-list.component.ts

```diff
@@ -181,6 +190,21 @@ export class AgendaListComponent extends BaseListViewComponent<ViewItem> impleme
         }
     }
 
+    /**
+     * Function to inspect all items. This will return the submitters of motions.
+     *
+     * @param item The view-item to inspect.
+     *
+     * @returns {string} All submitters from a motion as string separated by commas.
+     */
+    public getSubtitle(item: ViewItem): string {
```

Comment:
Move this as a generic to `ViewItem`. Let it be filled by the item repository. Let the return type be `string | null`, add a method `getAgendaSubtitle` to `IBaseViewModelWithAgendaItem` and default-implement it in `BaseViewModelWithAgendaItem` with just a `return null;`.